src/main/java/qna/domain/Answer.java

```diff
@@ -128,4 +129,16 @@ public boolean equals(Object o) {
     public int hashCode() {
         return Objects.hash(id, writer, question, contents, deleted);
     }
+
+    public DeleteHistory delete(User loginUser) throws CannotDeleteException {
+        validateAnswerIsEmpty(loginUser);
+        deleted = true;
+        return new DeleteHistory(ContentType.ANSWER, id, writer);
+    }
+
+    private void validateAnswerIsEmpty(User loginUser) throws CannotDeleteException {
+        if (!isOwner(loginUser)) {
```

Comment:
```suggestion
        if (isNotOwner(loginUser)) {
```
이렇게 써보는건 어떨까요? :)
메서드 네이밍 관련해서 [참고](https://velog.io/@hellojihyoung/%EA%B0%9C%EB%B0%9C-%EB%8F%84%EC%84%9C-Clean-Code-17%EC%9E%A5-%EB%83%84%EC%83%88%EC%99%80-%ED%9C%B4%EB%A6%AC%EC%8A%A4%ED%8B%B1#-g29-%EB%B6%80%EC%A0%95-%EC%A1%B0%EA%B1%B4%EC%9D%80-%ED%94%BC%ED%95%98%EB%9D%BC:~:text=%EB%B6%80%EC%A0%95%20%EC%A1%B0%EA%B1%B4%EC%9D%80%20%EA%B8%8D%EC%A0%95%20%EC%A1%B0%EA%B1%B4%EB%B3%B4%EB%8B%A4%20%EC%9D%B4%ED%95%B4%ED%95%98%EA%B8%B0%20%EC%96%B4%EB%A0%B5%EB%8B%A4.%20%EA%B0%80%EB%8A%A5%ED%95%98%EB%A9%B4%20%EA%B8%8D%EC%A0%95%20%EC%A1%B0%EA%B1%B4%EC%9D%84%20%ED%91%9C%ED%98%84%ED%95%9C%EB%8B%A4.) 해보시면 좋을 것 같아요~!

---

추가로 getter, setter 등 작성 된 메서드중에 사용되지 않는 메서드들은 삭제하는게 좋을 것 같아요 :)